src/app/components/team/Statuses/EditStatusDialog.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import { SliderPicker } from 'react-color';
+import styled from 'styled-components';
+import globalStrings from '../../../globalStrings';
+import { caption, opaqueBlack38, opaqueBlack87, units } from '../../../styles/js/shared';
+
+const maxLength = 35;
+
+const StyledColorPickerContainer = styled.div`
+  margin: ${units(2)} 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const StyledSelectedColorDisplay = styled.div`
+  width: ${units(5)};
+  height: ${units(5)};
+  border-radius: 50%;
+  background-color: ${props => props.color};
+`;
+
+const StyledSelectedColor = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+const StyledParamName = styled.span`
+  font: ${caption};
+  color: ${opaqueBlack38};
+`;
+const StyledHexValue = styled.span`
+  font: ${caption};
+  color: ${opaqueBlack87};
+`;
+
+const StyledSliderContainer = styled.div`
+  width: 80%;
+`;
```

Comment:
We agreed to use `makeStyles` instead of styled components from now on, including using `theme.spacing` instead of `units`.

Reply:
Also, if there is a good reason to use styled components instead of `makeStyles`, let's discuss!

Reply:
Sorry, I might have misunderstood. I thought we agreed either `makeStyles` or styled components were fine and we were just dropping inline styles.

Reply:
A good reason to use them is they are more straightforward to use and read, imo